### ISSUE TITLE
Data not recorded

### ISSUE THREAD

**Issue by upelectric** [tillsteinbach/VWsFriend#442]
Hi, I have a mysterious behaviour. I have two cars connected to vwsfriend. Everything works Fine for the GTE, but for my e-up I can overview, charges and live status in Grafana. All others like locations, trips, mileage, … shows no information. Only empty tables or graphs. I also can’t see the charge Power for the e-up as I can see it for the GTE. The car sends the parking position and it is also shown in vwsfriend, but no trips are recorded or locations. Vwsfriend version is 0.23.7 but I had the problem also with 0.23.3./5/6
I restarted it but no change. I can also see everything in the WC app. Do you have any idea?

**Comment by tillsteinbach** [comment]
hmmm, when you look on the status page in the backend: http://{IP}:4000/status/vehicle/{VIN}/ what information do you get for the e-up? My suspicion is that the API does not send all information for the e-up as it is using another vehicle platform.
Also: Do you see any warnings in the log?

**Comment by upelectric** [comment]
No, there are no warnings. I have the info in the log that the car provides the parking position and trips can be recorded but no trips are recorded. The info page ofvthe car shows range status, battery status, charging status, lots others and also maintenance status, odometer and the right parking position. But on the grafana mileage graph it shows nothing. The only difference is that I connected the e-up with ABRP and the GTE not. Could that be the problem?

**Comment by tillsteinbach** [comment]
No. Hmmm, I what does the parking position give when the ignition is on?

**Comment by upelectric** [comment]
I made some tests today. I removed the ABRP connection but no change. No trips recorded. If I drive with the GTE I can see an entry like „vehicle starts a trip“ and „vehicle trip ended“. If I Drive the e-up there is no such entry. But it updates data from WeConnect and the Live data in vwsfriend are fine. I also checked the status of the car. All data had a new timestamp but parkingPosition was outdated (1 hour old). The I created a test account on Tron…. and connected it to the e-up. Trips were recorded there exactly. Also in WeConnect the data are Fine including the parkingPosition. Are there some other logfiles so that I could check which data vwsfriend is receiving from WeConnect? I tried to increase the verbose mode but the I have tons of messages also from the GTE.

**Comment by tillsteinbach** [comment]
My guess is that the parking position is not removed when the car drives. Is that possible?

**Comment by upelectric** [comment]
How I can find it out? In the status tab of cars status in vwsfriend WebUI?

**Comment by tillsteinbach** [comment]
You can have a look in the status tab while the car is driving.

**Comment by upelectric** [comment]
Unfortunately it seems you are right. The parkingPosition is gone when the GTE is driving but it is still there for the e-up. Also the timestamps for mileage and parkingPosition on the status tab are 1 hour earlier than the others. For the GTE all are about the same.
Another difference is that the „fuel cap“ is always open at the e-up and not locked by ZV as it is on other cars. The e-up locks the charging cover like it would lock a charging cable. So I get a lot of wrong charging sessions.

**Comment by tillsteinbach** [comment]
I would guess that it would work to build some workarounds for the IDs. Unfortunately my private life went upside down from one day to the other. I don’t think that I can spend time on that in the near future.

**Comment by upelectric** [comment]
Hmm, ok. It seems I have to live without trips for the e-up

**Comment by byteschubser** [comment]
>  The I created a test account on Tron…. and connected it to the e-up. Trips were recorded there exactly. 

Please can you explain to me, what you've done? What is Tron? 

I have the same behaviour. As workarround I record trips manally in VWsFirend web interface
